src/directedness.jl

```diff
@@ -24,7 +26,7 @@ end
 @traitfn function arrange(
     ::MG, label_1, label_2, code_1, code_2
```

Comment:
We can further simplify by removing these `code_i` argument everywhere in the package

Reply:
I was thinking that too, but that meant small changes in many places. Done now!

Reply:
by the way: this simplification was done.
I also added a couple tests to check that the edge data was accessible and as expected.